app/javascript/components/mentoring/mentor_conversations_list.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(RelativeTime)
+
+function MentorConversationListRow({
+  trackIconUrl,
+  menteeAvatarUrl,
+  menteeHandle,
+  exerciseTitle,
+  isStarred,
+  haveMentoredPreviously,
+  isNewIteration,
+  postsCount,
+  updatedAt,
+  url,
+}) {
+  return (
+    <tr>
+      <td>
+        <img style={{ width: 100 }} src={trackIconUrl} />
+      </td>
```

Comment:
I'm trying to respond on the line above this, but the UI won't let me.

Add `alt`, for example, "icon indicating <track name>". If you don't want it to announce anything (e.g. presentation only), use `alt=""`

Reply:
Good catch! I've added the `alt` attributes.